evm/src/cpu/kernel/asm/core/terminate.asm

```diff
@@ -1,23 +1,57 @@
 // Handlers for operations which terminate the current context, namely STOP,
 // RETURN, SELFDESTRUCT, REVERT, and exceptions such as stack underflow.
 
+// Set parent context's CTX_METADATA_RETURNDATA_SIZE to 0.
+%macro set_parent_returndata_size_zero
+    // stack: (empty)
```

Comment:
I think it would be nice to have a `mstore_parent_context_metadata` macro, since we do this in a few places

Reply:
Good idea, I added two macros `mstore_parent_context_metadata(field), mstore_parent_context_metadata(field, value)` in `metadata.asm`.